### ISSUE TITLE
configparser fails

### ISSUE THREAD

**Issue by alphapapa** [ids1024/wikicurses#1]
I just installed from the latest commit and get this error:

```
$ wikicurses
Traceback (most recent call last):
  File "/usr/lib/python3.4/configparser.py", line 1116, in _unify_values
    sectiondict = self._sections[section]
KeyError: 'general'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/wikicurses", line 5, in <module>
    import wikicurses.ui as ui
  File "/usr/local/lib/python3.4/dist-packages/wikicurses/ui.py", line 7, in <module>
    from wikicurses.wiki import Wiki, WikiError
  File "/usr/local/lib/python3.4/dist-packages/wikicurses/wiki.py", line 10, in <module>
    from wikicurses.htmlparse import parseExtract, parseFeature
  File "/usr/local/lib/python3.4/dist-packages/wikicurses/htmlparse.py", line 13, in <module>
    if conf.getboolean('general', 'hide_references'):
  File "/usr/lib/python3.4/configparser.py", line 802, in getboolean
    raw=raw, vars=vars)
  File "/usr/lib/python3.4/configparser.py", line 776, in _get
    return conv(self.get(section, option, **kwargs))
  File "/usr/lib/python3.4/configparser.py", line 754, in get
    d = self._unify_values(section, vars)
  File "/usr/lib/python3.4/configparser.py", line 1119, in _unify_values
    raise NoSectionError(section)
configparser.NoSectionError: No section: 'general'
```

**Comment by ids1024** [comment]
You probably have an out of date `wikicurses.conf` file in `/etc`.  That said, it should probably have a default if that is not set in the configuration.

**Comment by ids1024** [comment]
This should be fixed by commit 6c310b7cec1b7435a924bf8b104a83ca2c45e76d.
